papers/doc/7-systems/k007232.md

```diff
@@ -0,0 +1,11 @@
+# Konami K007232
+
+a Konami's 2 channel PCM sound chip was used in their arcade boards in 1986-1990.
```

Comment:
`a Konami's 2 channel PCM sound chip was used in their arcade boards in 1986-1990.`

should be

`a 2-channel PCM sound chip from Konami which was used in some of their 1986-1990 arcade boards.`